python/py-agate-excel/Portfile

```diff
@@ -21,7 +21,7 @@ checksums           rmd160  3afd4d0df268d1742b5248b105a9c206171aea35 \
                     sha256  62315708433108772f7f610ca769996b468a4ead380076dbaf6ffe262831b153 \
                     size    161131
 
-python.versions     38 39 310 311
+python.versions     38 39 310 311 312
```

Comment:
While it looks like the current version of this package supports 3.12, version 0.2.5 only handled up to python3.9. Might be worth upgrading everything in general, versus just adding old versions to python3.12.